### ISSUE TITLE
stderr not implemented

### ISSUE THREAD

**Issue by rmx90210** [calvinmetcalf/node-process-es6#2]
I'm trying to use this with a package that calls `process.stderr.fd` and it's throwing an error in the browser:

    Uncaught TypeError: Cannot read property 'fd' of undefined

Does anyone know of a way to work around this?

Thanks!

**Comment by calvinmetcalf** [comment]
those things don't exist in the browser and can't be mocked without bringing in streams

**Comment by rmx90210** [comment]
Right, I figured there wasn't much a browser could do to mimic `stderr`.

How does `rollup-plugin-node-builtins` work in that case, if it contains code that can't run in the browser?

**Comment by calvinmetcalf** [comment]
it shouldn't contain code that can't run in the browser

**Comment by 4nte** [comment]
I'm having the same issue with my rollup config. 
@rbrtmrtn any progress?

**Comment by spike886** [comment]
I have the same issue
